hack/verify-generated-json-codecs.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
-echo "===== Verifying Generated Ugorji JSON Codecs ====="
+function cleanup() {
+    return_code=$?
+    os::test::junit::generate_report
+    os::util::describe_return_code "${return_code}"
+    exit "${return_code}"
+}
+trap "cleanup" EXIT
 
-output=$(find ${OS_ROOT}/vendor/k8s.io/kubernetes -name "types.generated.go")
-if [[ -n "${output}" ]]; then
-  os::log::fatal "FAILURE: Verification of existing ugorji JSON codecs failed. These should NOT exist:\n${output}"
-fi
+os::test::junit::declare_suite_start "verify/codecs"
```

Comment:
would prefer to match the verfiy-* name and the string here

Reply:
`generated` is always useless -- I'll do `json-codecs`

Reply:
the suite name should match the update script (if we already loose the error strings). I don't want to guess the necessary update script. Will you at least see the verify script name in the output?

Reply:
If the later is true, I would be fine with it.

Reply:
Script name will be in the output -- here is an example from the tests:

```
Running hack/verify-generated-deep-copies.sh:13: executing '/go/src/github.com/openshift/origin/hack/update-generated-deep-copies.sh --verify-only' expecting success...
SUCCESS after 16.514s: hack/verify-generated-deep-copies.sh:13: executing '/go/src/github.com/openshift/origin/hack/update-generated-deep-copies.sh --verify-only' expecting success
[INFO] hack/verify-generated-deep-copies.sh exited with code 0 after 00h 00m 17s
```